samples/golang/mywebapi/src/app/hello.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+)
+
+func sayHelloHandler(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprintf(w, "Hello")
+}
+
+func pingHandler(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprintf(w, "Ping")
+}
+
+func main() {
+	fmt.Println("Hello World!")
+	http.HandleFunc("/", sayHelloHandler)
+	http.HandleFunc("/api/ping", pingHandler)
+	err := http.ListenAndServe(":80", nil)
```

Comment:
Does this respond appropriately to SIGTERM and SIGINT signals? Test it by running in Docker locally and issuing signal calls to it. If it takes 10+ seconds to shut down, it means it didn't respond and will be slow to terminate in Kubernetes.

Reply:
I tried it and it takes somewhat 5-8 seconds to shut down.

Reply:
Hmm. It shouldn't take that long if it's correctly responding to signals. But it seems like it is responding to the signals, so that's good enough.